src/main/java/org/eolang/jeo/representation/XmirRepresentation.java

```diff
@@ -150,7 +150,7 @@ public String name() {
     public Bytecode toBytecode() {
         final Node xmir = this.xml.value();
         try {
-            new StrictXML(new XMLDocument(new XMLDocument(xmir).toString())).validate();
+            new StrictXmir(new XMLDocument(new XMLDocument(xmir).toString())).inner();
             return new XmlProgram(xmir).bytecode().bytecode();
```

Comment:
@volodya-lombrozo you can do here just
```java
new XmlProgram(new StrictXmir(new XMLDocument(xmir)).inner()).bytecode().bytecode()
```

Reply:
@maxonfjvipon I can't remove extra `new XMLDocument`. It's one more bug in `jcabi-xml`. You can try to remove it and you will see that unit tests fail.